service/history/configs/config.go

```diff
@@ -397,7 +398,8 @@ func NewConfig(
 		QueuePendingTaskMaxCount:         dc.GetIntProperty(dynamicconfig.QueuePendingTaskMaxCount, 10000),
 		QueueMaxReaderCount:              dc.GetIntProperty(dynamicconfig.QueueMaxReaderCount, 2),
 
-		TaskDLQEnabled: dc.GetBoolProperty(dynamicconfig.HistoryTaskDLQEnabled, true),
+		TaskDLQEnabled:                 dc.GetBoolProperty(dynamicconfig.HistoryTaskDLQEnabled, true),
+		AttemptsBeforeSendingTaskToDLQ: dc.GetIntProperty(dynamicconfig.HistoryAttemptsBeforeSendingTaskToDlq, 100),
```

Comment:
What is a good default value for this?

Reply:
yeah I think 100 is good. It will be reached after ~2.5 hours.